dataland-batch-manager/src/main/resources/application-development.properties

```diff
@@ -1,5 +1,6 @@
 dataland.dataland-batch-manager.client-id=dataland-batch-manager
 dataland.dataland-batch-manager.client-secret=password
 dataland.dataland-batch-managet.get-all-gleif-companies.force=false
+dataland.dataland-batch-managet.get-all-gleif-companies.flag-file=./this-file-does-not-exist
```

Comment:
Is it not enough to just not set the flag file?

Reply:
Good to know that an error occurs in that case and that it is not set to an empty string like done in the docker-compose.yml or the github workflow yamls.